common:
  msg_source: 2                                         #0: not use Lidar
                                                        #1: packet message comes from online Lidar
                                                        #2: packet message comes from Pcap file
  send_point_cloud_ros: true                            #true: Send point cloud through ROS or ROS2
lidar:
  - driver:
      lidar_type: IFW192S                       #LiDAR type - IFW192S IFW192A
      point_type: 0                             #point cloud type 0(x,y,z,intensity,ring) 1(x,y,z,intensity,ring,time)
      cloud_port: 8080                          #cloud port of lidar (pc port)
      check_local_port: 2367                    #check param of lidar device UDP (pc port)
      check_lidar_ip: 192.168.1.10              #check param of lidar device UDP (lidar ip)
      check_lidar_port: 2368                    #check param of lidar device UDP (lidar port)
      h_start_angle: -360                       #herizon Start angle of point cloud
      h_end_angle: 360                          #herizon End angle of point cloud 
      v_start_angle: -50                        #vertical Start angle of point cloud
      v_end_angle: 50                           #vertical End angle of point cloud
      min_distance: 0.02                        #Minimum distance of point cloud
      max_distance: 200                         #Maximum distance of point cloud
      pcap_repeat: true                         #True--repeat play pcap file
      is_device_load_calibration: false         #True--Use the lidar check param from lidar device
      calibrate_folder: /home/wenjie/project/driver/ws/src/inno_lidar_ros/config/ifw192s #
      pcap_file: /home/wenjie/project/pcap/ifw192s/1234.pcap #The path of pcap file

    ros:
      ros_frame_id: innolidar                             #Frame id of packet message and point cloud message
      ros_send_point_cloud_topic: /innolidar_points       #Topic used to send point cloud through ROS
      ros_send_imu_topic: /inno_imu       #Topic used to send point cloud through ROS
